django/db/models/base.py

```diff
@@ -494,6 +494,10 @@ def __getstate__(self):
         state.pop("peers", None)
         return state
 
+    def __del__(self):
+        if hasattr(self, "_fields_cache"):
+            del self._fields_cache
```

Comment:
My question was getting at, under what conditions would `_fields_cache` be missing?